src/test/java/org/kohsuke/github/AbstractGHAppInstallationTest.java

```diff
@@ -13,17 +16,25 @@
 import java.security.KeyFactory;
 import java.security.PrivateKey;
 import java.security.spec.PKCS8EncodedKeySpec;
+import java.text.ParseException;
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
 import java.util.Base64;
 import java.util.Date;
+import java.util.List;
+import java.util.Set;
 
 // TODO: Auto-generated Javadoc
 /**
  * The Class AbstractGHAppInstallationTest.
  */
 public class AbstractGHAppInstallationTest extends AbstractGitHubWireMockTest {
 
+    private static String ENV_GITHUB_APP_ID = "GITHUB_APP_ID";
```

Comment:
The default GHApps have no marketplace listing. Hence, I had to rely on my own app listing.